ruma-client-api/src/r0/capabilities.rs

```diff
@@ -1,3 +1,129 @@
 //! Endpoints for querying the server's supported feature set
 
+use std::{borrow::Cow, collections::BTreeMap};
+
+use serde::{Deserialize, Serialize};
+use serde_json::{from_value as from_json_value, to_value as to_json_value, Value as JsonValue};
+
+use get_capabilities::{ChangePasswordCapability, RoomVersionsCapability};
```

Comment:
Sorry for not specifying, but the individual capabilities should also move to this module as opposed to being imported from `get_capabilities`.